spec/worker_black_box_spec.rb

```diff
@@ -277,5 +277,70 @@
         assert_openshift_objects_data(sources_data[:azure]['7']) # added
       end
     end
+
+    context "when the image changes" do
+      before do
+        # Fill openshift first
+        #
+        # 1st sync
+        #
+        @sources_in_openshift = [topological_sources[:openshift]['1'],
+                                 topological_sources[:openshift]['2'],
+                                 topological_sources[:amazon]['4'],
+                                 topological_sources[:amazon]['5']]
+        @available_sources = [available_sources_data(:openshift)['1'],
+                              available_sources_data(:openshift)['2'],
+                              available_sources_data(:amazon)['4'],
+                              available_sources_data(:amazon)['5']].compact
+
+        stub_api_init(:application_types => application_types_response,
+                      :applications      => applications_response,
+                      :source_types      => source_types_response,
+                      :tenants           => tenants_response,
+                      :topology_sources  => list(@sources_in_openshift))
+
+        init_config(:openshift => 1, :amazon => 1, :azure => 1) # per collector
+
+        stub_api_source_calls(sources_data[:openshift]['1'])
+        stub_api_source_calls(sources_data[:openshift]['2'])
+        stub_api_source_calls(sources_data[:amazon]['4'])
+        stub_api_source_calls(sources_data[:amazon]['5'])
+
+        # Run orchestrator
+        subject.send(:make_openshift_match_database)
+        expect(@added).to eq(@available_sources.size)
+
+        assert_openshift_objects_count(1 + 2) # 1 for openshift, 2 for amazon
+      end
+
+      it "updates the deployment config" do
+        # Set the image so that it changes the deployment config
+        object_manager.instance_variable_set(:@collector_image, "quay.io/ansible:thingything")
+
+        #
+        # 2nd sync
+        #
+        # added 1 azure + 1 amazon, deleted 1 openshift + 1 amazon
+        sources_from_api = [topological_sources[:openshift]['1'],
+                            topological_sources[:amazon]['4'],
+                            topological_sources[:amazon]['6'], # new
+                            topological_sources[:azure]['7']]  # new
+
+        # :application_types request cached
+        stub_api_init(:applications      => applications_response,
+                      :source_types      => source_types_response,
+                      :tenants           => tenants_response,
+                      :topology_sources  => list(sources_from_api))
```

Comment:
There are two spaces before the hash rocket which is causing the rubocop error.  Just need to remove one and align the lines above.